src/App.tsx

```diff
@@ -1,39 +1,72 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Props = any;
```

Comment:
Don't do so, unless you have a strong reason to.
By using `any` you effectively switch off all typescript checks

In your case please, look for alternatives:
1. Use `unknown` or `never`
2. Just remove typisation from `App` and let typescript to calculate all types for you.